Gemfile

```diff
@@ -1,3 +1,6 @@
 source 'https://rubygems.org'
 
+# dev dependency
+gem 'pedicel-pay', git: 'https://github.com/cub8/pedicel-pay', branch: 'openssl-30-support'
+
```

Comment:
```suggestion
```

I've released version 0.0.8 of pedicel-pay based on https://github.com/clearhaus/pedicel-pay/pull/16.

Reply:
Removed this dependency.